hivemq-edge/src/main/java/com/hivemq/protocols/ProtocolAdapterWrapper.java

```diff
@@ -29,7 +29,7 @@
 import com.hivemq.extension.sdk.api.annotations.NotNull;
 import com.hivemq.extension.sdk.api.annotations.Nullable;
 
-public class ProtocolAdapterWrapper<T extends ProtocolAdapter> implements ProtocolAdapter {
```

Comment:
nice, was a big mistake to make it an ProtocolAdapter. Was easy at the time, but super error prone